### ISSUE TITLE
HTTP 500

### ISSUE THREAD

**Issue by whitefi** [programmerben/KodiKontrol#5]
Can't seem to get past the internal http 500 error.

**Comment by smiley6180** [comment]
Hello, were you able to fix this issue? I am getting this;
GET /sickrage 500 5.808 ms - 12
GET /favicon.ico 500 1.425 ms - 12

**Comment by whitefi** [comment]
Yes, I was getting the 500 error as I was testing in my browser. That won't
work.
The request must come from Amazon's servers. The issue was the certificate
I was using did not have my public IP in the certificates subject
alternative name.

**Comment by smiley6180** [comment]
Perfect, got it working. thank you.
